### DISCUSSION TITLE
Excel download in vapor says "Failed - Server issue"

### DISCUSSION THREAD

**Discussion by Rik0253** [laravel/nova-issues#4073] [Q&A]
- Laravel Version: 8.51
- Nova Version: *
- PHP Version: 8.0
- maatwebsite/laravel-nova-excel: 1.2

### Detailed steps to reproduce the issue :
1. When I try to download excel in nova in vapor server it gives error . The file does not download and says Failed- Server issue

Can anyone suggest what to do as it works in local.

**Comment by gepopp** [comment]
Hi there,
i had the same issue bcs i was using a CORS Middleware, mby worth a try setting all yout middleware back to the defaults and see if it works.

**Reply by Rik0253** [reply]
Hi,
I haven't used CORS Middleware then also it gives the above error

**Comment by Rik0253** [comment]
> * Laravel Version: 8.51
> * Nova Version: *
> * PHP Version: 8.0
> * maatwebsite/laravel-nova-excel: 1.2
> 
> ### Description:
> ### Detailed steps to reproduce the issue :
> 1. When I try to download excel in nova in vapor server it gives error . The file does not download and says Failed- Server issue
> 
> Can anyone suggest what to do as it works in local.

Can anyone help on this ?
